discover/faq.md

```diff
@@ -341,3 +341,28 @@ veFeeDistributor has a start\_time of 1663804800 (Thu Sep 22 2022 00:00:00).
 You can publish Fixed Price Assets to any chain that Ocean supports.
 
 </details>
+
+<details>
+
+  <summary>What psdnOCEAN is and how it differs from OCEAN?</summary>
+
+OCEAN is the main currency of the Ocean network, particularly the Ocean Market. psdnOCEAN is a staking token used for data farming network rewards. By converting OCEAN to psdnOCEAN, users can earn passive staking rewards. psdnOCEAN is not directly affiliated with Ocean Protocol and is operated by an independent entity named H20. Locking contracts for H20 and psdnOCEAN are open source and can be examined publicly. To get psdnOCEAN, users lock their OCEAN and then receive an equivalent amount of psdnOCEAN in return. Although the locking and staking contracts have been audited, users should do their own research and put at risk only the amounts that they are willing to lose.
+
+
+</details>
+
+<details>
```

Comment:
These 2 questions should be in the general section

Reply:
done